tasks/build-binary-new-cflinuxfs4/builder.rb

```diff
@@ -127,6 +127,41 @@ def strip_top_level_directory_from_zip(filename, destination)
   end
 end
 
+module DependencyBuildHelper
+  class << self
+    def build_nginx_helper(source_input, custom_options, static=false)
+      artifacts = "#{Dir.pwd}/artifacts"
+      destdir = Dir.mktmpdir
+      Dir.mktmpdir do |dir|
+        Dir.chdir(dir) do
+          Runner.run('wget', source_input.url)
+          # TODO validate pgp
```

Comment:
You forgot to implement this validation.

Reply:
This was part of the code I took from older `builder.rb`. I don't know what `pgp` means, do you have any context on this?

Ref: https://github.com/cloudfoundry/buildpacks-ci/blob/master/tasks/build-binary-new/builder.rb#L326

Reply:
@TisVictress After some quick investigation I found what is PGP an how we can validate the file integrity with it. I update the PR with the changes